invenio_base/cli.py

```diff
@@ -85,7 +85,8 @@ def migrate_secret_key(old_key):
         except Exception:
             current_app.logger.error(
                 'Failed to initialize entry point: {0}'.format(ep))
-            raise
+            raise click.ClickException(
```

Comment:
This will cause a new exception to be raised, which will mask the existing exception, and thus the origianl reason why the initialization failed (e.g. you won't have the stacktrace).